starter/script.js

```diff
@@ -1 +1,78 @@
+        alert("Mr. Krabs believed your lie about fending off a nematode infestation and excused your tardiness");
+      } else if (question2b === 'TRUTH') {
+        alert("Mr. Krabs appreciates your honestly and will only punish you by not paying you for the day");
+      } else if (question3b === 'NO'){
+        alert("Mr. Krabs knows you're lying and he doesn't care what your reason is, HE NEEDS YOU TO MAKE HIM MONEY");
+      } else {
+        alert("ne of your responses was not valid");
+      }
+    break;
+
+    default:
+    alert("Please type a valid response");
+  }
+}
+
+function plankton() {
+
+  var question = prompt("You're plotting to steal the Krabby Patty formula again but you have your restaurant, The Chum Bucket, to run.\nDo you attempt to STEAL the formula again or are you going to try to MANAGE your own restaurant").toUpperCase();
+
+  switch(question) {
+
+    case 'STEAL':
+      var question2a = prompt("You have three top-secret plan: do you choose PLAN A, PLAN Z, or PLAN E: The Most Evilest Evil Plan of All Time").toUpperCase();
+      if (question2a === "PLAN E") {var question2a = prompt("So PLAN E is still in the work and you haven't come up with anything, so you'll just have to choose between PLAN A and PLAN Z for now").toUpperCase();}
+      var question3a = prompt("The plan must be executed correctly to work, can you follow directions well? YES or NO").toUpperCase();
+      if (question2a === 'PLAN A' && question3a === 'YES') {
+        alert("You send a robotic Mr. Krabs in and successfully convinced SpongeBob to show you where the formula is!");
+      } else if (question2a === 'PLAN Z' && question3a === 'YES') {
+        alert("You were able to trick Mr. Krabs in giving you the formula by bribing him with fake gold doubloons!");
+      } else if (question3a === 'NO'){
+        alert("Things did not go according to plans cuz you can't follow directions. YA DONE GOOF UP");
+      } else {
+        alert("One of your responses was not valid")
+      }
+    break;
+
+    case 'MANAGE':
+      var question2b = prompt("Well you first need some employees, you have two possible candidate to choose from: PATRICK or SQUIDWARD").toUpperCase();
+      var question3b = prompt("How much are you going to pay them? (enter an integer)");
+      if (question2b === 'PATRICK') {
+        alert("Patrick is so dumb that he doesn't care how much you pay him, he'll accept the job anyway");
+      } else if (question2b === 'SQUIDWARD' && question3b < 9) {
+        alert("Squidward thinks he deserves to be paid more than that, guess you're stuck with Patrick");
+      } else if (question2b === 'SQUIDWARD' && question3b > 9){
+        alert("Squidward begrudingly accepts your offer, at least you pay more than the Krusty Krabs");
+      } else {
+        alert("One of your responses was not valid")
+      }
+    break;
+
+    default:
+    alert("Please type a valid response");
+  }
+}
```

Comment:
I thought it was interesting approach that you wrapped a bunch of conditionals inside of a switch, although stylistically this is not preferable. You could instead break the conditional statements out into separate functions and call those functions accordingly. Great work overall! 👍